eng/scripts/Update-Mgmt-Yml.ps1

```diff
@@ -38,7 +38,10 @@ $PrIncludes = New-Object "System.Collections.Generic.List[String]"
 
 $MgmtDirs = Get-ChildItem -Path "$PackagesPath" -Directory -Recurse -Depth 1 | Where-Object { $_.FullName -match "(.Microsoft\.Azure(Stack)?\.Management.)|(.\\mgmt)" }
```

Comment:
```suggestion
$MgmtDirs = (Get-ChildItem -Path "$PackagesPath" -Directory -Recurse -Depth 1).Where({$_.FullName -match "(.Microsoft\.Azure(Stack)?\.Management.)|(.\\mgmt)"})
```

Reply:
:) I like the suggestion but I think that is for another clean-up PR.